keyboards/scrabblepad/keymaps/random/keymap.c

```diff
@@ -0,0 +1,75 @@
+#include "scrabblepad.h"
+#include <stdlib.h>
+
+static uint16_t keystroke;
+static bool shift_pressed;
+
+static int lower = (int) KC_A;
+static int upper = (int) KC_Z;
+
+uint16_t random_keycode(void) {
+  return lower + (rand() % (upper - lower + 1));
+};
+
+enum custom_keycodes {
+  RND_KEY = SAFE_RANGE
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch(keycode) {
+    case RND_KEY:
+      if (record->event.pressed) {
+        keystroke = random_keycode();
```

Comment:
there is a function somewhere for a random keypress

Reply:
https://github.com/qmk/qmk_firmware/blob/master/quantum/quantum.c#L914
`tap_random_base64`